examples/Cassie/dispatcher_robot_in.cc

```diff
@@ -130,31 +162,63 @@ int do_main(int argc, char* argv[]) {
 
   // Create and connect LCM command echo to network
   auto net_command_sender = builder.AddSystem<RobotCommandSender>(plant);
-  auto net_command_pub =
-      builder.AddSystem(LcmPublisherSystem::Make<dairlib::lcmt_robot_input>(
-          "NETWORK_CASSIE_INPUT", &lcm_network, {TriggerType::kPeriodic},
-          FLAGS_pub_rate));
+
+  LcmPublisherSystem* command_pub;
+  if (FLAGS_sim) {
+    command_pub =
+        builder.AddSystem(LcmPublisherSystem::Make<dairlib::lcmt_robot_input>(
+            "CASSIE_INPUT", &lcm_local, {TriggerType::kForced}));
+  } else {
+    command_pub =
+        builder.AddSystem(LcmPublisherSystem::Make<dairlib::lcmt_robot_input>(
+            "NETWORK_CASSIE_INPUT", &lcm_network, {TriggerType::kPeriodic},
+            FLAGS_pub_rate));
+  }
 
   builder.Connect(input_supervisor->get_output_port_command(),
                   net_command_sender->get_input_port(0));
 
-  builder.Connect(*net_command_sender, *net_command_pub);
+  builder.Connect(*net_command_sender, *command_pub);
 
   // Finish building the diagram
   auto owned_diagram = builder.Build();
   owned_diagram->set_name("dispatcher_robot_in");
 
   // Channel names of the controllers
-  std::vector<std::string> input_channels;
-  input_channels.push_back(FLAGS_control_channel_name_1);
-  input_channels.push_back(FLAGS_control_channel_name_2);
-  input_channels.push_back(FLAGS_control_channel_name_3);
+  std::vector<std::string> input_channels = {FLAGS_control_channel_name_initial,
+                                             "PD_CONTROL",
+                                             "OSC_STANDING",
+                                             "OSC_WALKING",
+                                             "OSC_JUMPING",
+                                             "OSC_RUNNING"};
+  if (!FLAGS_control_channel_name_additional.empty()) {
+    input_channels.push_back(FLAGS_control_channel_name_additional);
+  }
 
   // Run lcm-driven simulation
-  systems::LcmDrivenLoop<dairlib::lcmt_robot_input,
+  CassieLcmDrivenLoop<dairlib::lcmt_robot_input,
                          dairlib::lcmt_controller_switch>
       loop(&lcm_local, std::move(owned_diagram), command_receiver,
-           input_channels, FLAGS_control_channel_name_1, switch_channel, true);
+           input_channels, FLAGS_control_channel_name_initial, switch_channel,
+           true, FLAGS_state_channel_name);
+
+  auto msg = dairlib::lcmt_pd_config();
+  msg.timestamp = 0;
+  msg.num_joints = 10;
+  msg.joint_names = {"hip_roll_left_motor",  "hip_roll_right_motor",
+                     "hip_yaw_left_motor",   "hip_yaw_right_motor",
+                     "hip_pitch_left_motor", "hip_pitch_right_motor",
+                     "knee_left_motor",      "knee_right_motor",
+                     "toe_left_motor",       "toe_right_motor"};
+  msg.desired_position = {-0.01, .01, 0, 0, 0.55, 0.55, -1.7, -1.7, -1.8, -1.8};
```

Comment:
We need to change this to not assume a hardcoded joint order before merging in the drake joint reorder